src/sentry/assistant/guides.py

```diff
@@ -110,4 +110,18 @@
             }
         ],
     },
+    "dynamic_counts": {
+        "id": 4,
+        "required_targets": ["dynamic_counts"],
+        "steps": [
+            {
+                "title": _("These counts have changed"),
+                "message": _(
+                    "These numbers and the bar chart now respect the time selected and any"
+                    "search filters you've applied. You can hover to see the totals."
+                ),
+                "target": "dynamic_counts",
+            }
+        ],
```

Comment:
I don't think you need the steps/title/message in the server definition of guides. It doesn't get used in the API responses. The other guides need to be cleaned-up still. I can do that separately.

Reply:
Ok I'll remove them from here